src/declarative.js

```diff
@@ -107,7 +107,7 @@
     return moduleRecords[key] || (moduleRecords[key] = {
       key: key,
       dependencies: [],
-      module: new Module({}),
+      module: new Reflect.Module({}),
```

Comment:
I would prefer having these add-ons as modules :-) `Module()` is surely left over from the old spec, and I think using `Reflect.Module` makes it clearer what's going on here, though I agree it probably doesn't matter much in practice.